source/mainarrays.f90

```diff
@@ -56,9 +54,7 @@ subroutine allocate_arrays
 #ifdef CR
   call cr_expand_arrays(npart)
 #endif
-#ifdef COLLIMAT
   call collimation_allocate_arrays
-#endif
```

Comment:
Hmm, this is not a good idea. Collimation allocates a lot of arrays. Should probably find a way around this.

Reply:
@kyrsjo There's still this ...

Reply:
Could we call that from inside the COLL block input parsing maybe?

Reply:
yeah, we could probably call the alloc routine with sizes 1 (if unallocated causes stuff to explode) and then call expand after a coll block has been seen. I.e. in daten when the openBlock flag is true.